include/net/nrf_cloud_pgps.h

```diff
@@ -66,6 +66,15 @@ struct nrf_cloud_pgps_prediction {
 	uint32_t sentinel;
 } __packed;
 
+/** Omit the prediction count from P-GPS request */
+#define NRF_CLOUD_PGPS_REQ_NO_COUNT	0
```

Comment:
I understand the intent here, but I'm not sure how the cloud side will behave when it receives some but not all of the parameters.

Reply:
Should be ok -- the REST API indicates that it uses sane defaults.

Reply:
Only the data obj is required.
Is no parameters are present, it uses the defaults.

Reply:
Yes.  I have not tested it with partial or no parameters, but assuming the values documented are in effect, it's fine.

Reply:
I have tested it